geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ConnectCommand.java

```diff
@@ -157,7 +158,23 @@ public Result connect(
       result = jmxConnect(gfProperties, useSsl, jmxManagerEndPoint, locatorEndPoint, false);
     }
 
-    return result;
+    OperationInvoker invoker = gfsh.getOperationInvoker();
+    if (!invoker.isConnected()) {
+      return result;
+    }
+
+    String gfshVersion = gfsh.getVersion();
+    try {
+      String version = invoker.getRemoteVersion();
```

Comment:
Another trivial nitpick, but given `gfshVersion` above, I might call this `memberVersion` or `remoteVersion` instead.